templates/relays.html

```diff
@@ -6,6 +6,12 @@
       // convert to utf8
       viewHexDataAs(element.id, "utf-8")
     })
+
+    {{ if len .RelaysInfoContainers }}
+      const relays = {{ .RelaysInfoContainers }}
+      activateTabbarSwitcher("poolTable", "poolTabs", `t${relays[0].Days}d`)
+    {{ end }}
+    activateTabbarSwitcher("blocksTable", "blockTabs", "latest_blocks")
```

Comment:
Question: Is it intended that there may be two `activateTabbarSwitcher` calls?

Reply:
Yes because there are two separate tabbars + content on that page